pkg/providers/imagefamily/azlinux.go

```diff
@@ -165,6 +165,7 @@ func (u AzureLinux) CustomScriptsNodeBootstrapping(
 	storageProfile string,
 	nodeBootstrappingClient types.NodeBootstrappingAPI,
 	fipsMode *v1beta1.FIPSMode,
+	localDNS *v1beta1.LocalDNS,
```

Comment:
@comtalyst is there not a cleaner way to do this without requiring flowing this to N image providers?